lib/expression.ex

```diff
@@ -120,11 +120,15 @@ defmodule Expression do
   end
 
   def evaluate_as_string!(expression, context \\ %{}, mod \\ Expression.Callbacks) do
```

Comment:
```suggestion
  def evaluate_as_string!(expression, context \\ %{}, mod \\ Expression.Callbacks)
  def evaluate_as_string!(nil, _context, mod), do: ""
  def evaluate_as_string!(expression, context, mod) do
```

Thanks @maryamaljanabi the change is correct but could we use pattern matching here?

Reply:
Thanks a lot, I'll make an update